apps/portals/src/portal-components/SynapseComponentCollapse.tsx

```diff
@@ -0,0 +1,72 @@
+import { SynapseConfig } from 'types/portal-config'
+import React, { useState } from 'react'
+import { Box, Collapse, SxProps, Typography } from '@mui/material'
+import {
+  KeyboardArrowDownTwoTone,
+  KeyboardArrowUpTwoTone,
+} from '@mui/icons-material'
+import { SynapseComponent } from '../SynapseComponent'
+
+export type SynapseComponentCollapseProps = {
+  text: string
+  synapseConfig: SynapseConfig
+  defaultVisible?: boolean // default to false (collapsed)
+}
+
+/**
+ * Wrap any Synapse config object in a collapse
+ * @param props
+ * @returns
+ */
+export default function SynapseComponentCollapse({
+  text,
+  synapseConfig,
+  defaultVisible,
+}: SynapseComponentCollapseProps) {
+  const [show, setShow] = useState(defaultVisible)
+  const iconSx: SxProps = {
+    color: 'grey.700',
+    marginBottom: '-5px !important',
+    height: '18px',
+  }
+
+  return (
+    <div className="MarkdownCollapse">
+      <Box
+        sx={{
+          display: 'flex',
+          justifyContent: 'space-between',
+          backgroundColor: 'grey.200',
+          padding: '15px',
+        }}
+      >
+        <Typography
+          variant="smallLink"
+          onClick={() => setShow(!show)}
+          aria-controls="collapse-text"
+          aria-expanded={show}
+          sx={{
+            '&:hover': {
+              cursor: 'pointer',
+            },
+          }}
+        >
+          {text}{' '}
+          {show ? (
+            <KeyboardArrowUpTwoTone sx={iconSx} />
+          ) : (
+            <KeyboardArrowDownTwoTone sx={iconSx} />
+          )}
+        </Typography>
```

Comment:
Looks good! Out of curiosity, how did you decide on using `Typography` (vs something like a `Button` with an `endIcon`)?